src/lib/dialog/dialog.scss

```diff
@@ -0,0 +1,52 @@
+@import '../core/style/elevation';
+@import '../a11y/_a11y';
+
+
+$md-dialog-padding: 24px !default;
+$md-dialog-border-radius: 2px !default;
+$md-dialog-max-width: 80vw !default;
+$md-dialog-max-height: 65vh !default;
+
+md-dialog-container {
+  @include md-elevation(24);
+
+  display: block;
+  padding: $md-dialog-padding;
+  border-radius: $md-dialog-border-radius;
+  box-sizing: border-box;
+  overflow: auto;
+  max-width: $md-dialog-max-width;
+
+  // The dialog container should completely fill its parent overlay element.
+  width: 100%;
+  height: 100%;
+
+  @include md-high-contrast {
+    outline: solid 1px;
+  }
+}
+
+md-dialog-content, [md-dialog-content] {
+  display: block;
+  margin: 0 $md-dialog-padding * -1;
+  padding: 0 $md-dialog-padding;
+  max-height: $md-dialog-max-height;
+  overflow: auto;
+}
+
+[md-dialog-title] {
+  font-size: rem(2);
```

Comment:
I'd like to avoid using this `rem` function for now... the fact that it's even here is a leftover from an earlier version. During beta we're going to work on a more comprehensive typography solution.